src/components/CreateVmWizard/CreateVmWizard.js

```diff
@@ -333,13 +334,16 @@ class CreateVmWizard extends React.Component {
             nics: !template
               ? []
               : template.get('nics', List())
-                .map(nic => ({
-                  id: nic.get('id'),
-                  name: nic.get('name'),
-                  vnicProfileId: nic.getIn(['vnicProfile', 'id']),
-                  deviceType: nic.get('interface'),
-                  isFromTemplate: true,
-                }))
+                .map(nic => {
+                  const vnicProfileId = nic.getIn(['vnicProfile', 'id'])
+                  return {
+                    id: nic.get('id'),
+                    name: nic.get('name'),
+                    vnicProfileId: !vnicProfileId ? EMPTY_VNIC_PROFILE_ID : vnicProfileId,
```

Comment:
Hi Scott, the return value is null not undefined.
I tried to use your suggestion but it wasn't work in that case: 
> Returns the value within the provided collection associated with the provided key, or notSetValue if the key is not defined in the collection.

quoted from immutableJS [documentation](https://immutable-js.github.io/immutable-js/docs/#/get).
Null considered as defined value, so we can't use the default value in this case,
If you have another suggestion I would like to change it.
Thanks :)

Reply:
Ah ok I forgot about that.  Humm.  Since `null` and `undefined` and `0` and `''` are considered truthy  false, this should work in all cases:

```jsx
   vnicProfileId: nic.getIn(['vnciProfile', 'id']) || EMPTY_VNIC_PROFILE_ID,
```

Reply:
Done :)